core/src/main/java/edu/softserve/zoo/model/Animal.java

```diff
@@ -0,0 +1,69 @@
+package edu.softserve.zoo.model;
+
+import java.time.LocalDate;
+
+public class Animal extends BaseEntity {
+    private String nickname;
+    private Species species;
+    private House house;
+    private LocalDate birthday;
+    private int temperatureMin;
```

Comment:
let's assume this field is empty in DB. which value will it be applied for this field? replace with wrapper type